moncli/entities/group.py

```diff
@@ -266,16 +269,27 @@ def add_item(self, item_name: str, *args, **kwargs):
             api_key=self.__creds.api_key_v2,
             group_id=self.id,
             **kwargs)
-        return en.Item(creds=self.__creds, **item_data)
-
-
-    def get_items(self, get_column_values: bool = True, *args, **kwargs):
+        items = en.Item(creds=self.__creds, **item_data)
+        if as_model:
+            if not issubclass(as_model, m.MondayModel):
+                raise ItemError(
```

Comment:
Similar to ItemError, please create GroupError with the same structure using 'Group' as the entity_name.